pkg/deployments/tasks/20221122-composable-stable-pool-v2/input.ts

```diff
@@ -3,12 +3,19 @@ import Task, { TaskMode } from '../../src/task';
 export type ComposableStablePoolDeployment = {
   Vault: string;
   ProtocolFeePercentagesProvider: string;
+  FactoryVersion: string;
+  PoolVersion: string;
 };
 
 const Vault = new Task('20210418-vault', TaskMode.READ_ONLY);
 const ProtocolFeePercentagesProvider = new Task('20220725-protocol-fee-percentages-provider', TaskMode.READ_ONLY);
 
+const BaseVersion = { version: 2, deployment: '20221122-composable-stable-pool-v2' };
```

Comment:
Seems like we should be able to pull the name and deployment fields from the deployment task automatically. This would avoid us having to check for consistency. (Although we should anyway)

Reply:
Agreed. Let's wait until #2039 to add that though.